pkg/controller/core.oam.dev/v1alpha2/application/assemble/testdata/filter_annotations.yaml

```diff
@@ -0,0 +1,189 @@
+        workload:
+          definition:
+            apiVersion: apps/v1
+            kind: Deployment
+      status: {}
+  components:
+    - raw:
+        apiVersion: core.oam.dev/v1alpha2
+        kind: Component
+        metadata:
+          labels:
+            app.oam.dev/component-revision-hash: 943da166b084e088
+          name: frontend
+        spec:
+          workload:
+            apiVersion: apps/v1
+            kind: Deployment
+            metadata:
+              labels:
+                app.oam.dev/appRevision: website-v1
+                app.oam.dev/component: frontend
+                app.oam.dev/name: website
+                workload.oam.dev/type: webservice
+            spec:
+              selector:
+                matchLabels:
+                  app.oam.dev/component: frontend
+              template:
+                metadata:
+                  labels:
+                    app.oam.dev/component: frontend
+                spec:
+                  containers:
+                    - image: nginx
+                      name: frontend
+                      ports:
+                        - containerPort: 80
+        status:
+          observedGeneration: 0
+  resourcesConfigMap:
+    name: website-v1
```

Comment:
please clean unrelated fields in this test data

Reply:
done